cookiecutter/generate.py

```diff
@@ -306,7 +306,7 @@ def generate_files(
         )
 
     with work_in(template_dir):
-        env.loader = FileSystemLoader('.')
+        env.loader = FileSystemLoader(['.', '../templates'])
```

Comment:
This should use os.path.sep instead so that it is compatible with windows and unix systems.